client/client-multi/client-cpp/kaa/KaaClientStateListener.hpp

```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  */
 
-#ifndef IKAACLIENTSTATELISTENER_HPP_
-#define IKAACLIENTSTATELISTENER_HPP_
+#ifndef KAACLIENTSTATELISTENER_HPP_
+#define KAACLIENTSTATELISTENER_HPP_
 
 #include <memory>
 
+#include "kaa/channel/IKaaChannelManager.hpp"
```

Comment:
Move EndpointConnectionInfo to a separate header in order to not to include IKaaChannelManager.hpp